public/app/screens/main/sections/devices/device-detail/device-detail.jsx

```diff
@@ -0,0 +1,106 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+var styleMixin = require('mixins/style-mixin');
+var roomStore = require('stores/room');
+var DeviceState = require('./../device-state');
+
+var DeviceDetail = React.createClass({
+  mixins: [
+    styleMixin(require('./style.scss')),
+    Router.State],
+
+
+  getInitialState: function getInitialState () {
+    var device = roomStore.getDevice(this.getParams().deviceId);
```

Comment:
We do NOT interact with directly. Please use action instead.
https://facebook.github.io/flux/